### ISSUE TITLE
WEBP to/from AVIF conversion

### ISSUE THREAD

**Issue by hamidpeywasti** [contao/contao#7688]
### Description

Hi,
There's conversion between JPG, PNG and PNG to/from WEBP and AVIF, Why not conversion between AVIF and WEBP?

**Comment by ausi** [comment]
There are endless possibilities for conversion combinations that we could add, so we tried to provide the options that might be used most often.

But as WEBP is supported by most browsers today, we could probably add it as a “base” format (like PNG and JPEG currently).

**Comment by hamidpeywasti** [comment]
Yes that's correct. In my opinion, instead of considering all possibilities or even best ones, there should be two sets of options; one for the **input formats** we want to be considered in conversion and one for the **output formats** we want to have on the website. In this way the user has more choices.

I had been using some images as **webp** format instead of jpeg and png and now I can't convert them to **avif** format for the website.
